kythe/typescript/indexer.ts

```diff
@@ -1598,8 +1649,9 @@ class Visitor {
     let declKw;
     if (ts.isVariableDeclaration(varDecl)) {
       declKw = initializerList.kind === ts.SyntaxKind.CatchClause ?
-          '(local var)' :
-          initializerList.flags & ts.NodeFlags.Const ? 'const' : 'let';
+                                                       '(local var)' :
+          initializerList.flags & ts.NodeFlags.Const ? 'const' :
+                                                       'let';
```

Comment:
It'd be nice if these pure reformattings (e.g. this and lines 1830+ below) were in a separate PR.

Reply:
https://github.com/kythe/kythe/pull/4842